docs/source/building/platforms/maxwell_desy.rst

```diff
@@ -42,14 +43,16 @@ submission script:
    #SBATCH --partition=<partition> # mpa # maxgpu # allgpu
    #SBATCH --time=01:00:00
    #SBATCH --nodes=1
-   #SBATCH --constraint=A100 # V100 # V100&GPUx2 # V100&GPUx4
+   #SBATCH --constraint=A100&GPUx4 # A100 # V100 # V100&GPUx2 # V100&GPUx4
```

Comment:
Maybe we should reflect the main architecture there?
```suggestion
   #SBATCH --constraint=A100&GPUx4 # A100&GPUx1
```

Reply:
Agreed, that is more reasonable.